cookbook/setup_data/add_translation_packs.rst

```diff
@@ -0,0 +1,25 @@
+
+How to Add Translation Packs
+============================
+
+.. _Crowdin: http://crowdin.net/project/akeneo/
+
+
+Akeneo PIM UI is translated through `Crowdin`_ (feel free to contribute!).
+
+Once a week week, new translation keys are pushed to Crowdin, and new validated translations are pulled to our Github repository.
+
+Akeneo PIM contains translation packs for all languages with more than 80% of translated keys.
```

Comment:
maybe add a link to http://docs.akeneo.com/latest/cookbook/localization/change_pim_locale.html#how-to-allow-more-locales